publication/ver11/societal_impact.md

```diff
@@ -0,0 +1,239 @@
+or to provide an alternative physical interface to some digital functionality
+(which provide opportunities to improve accessibilty with custom interfaces).
+
+Specifically, the WoT standards provide consistent metadata for the access of 
+IoT systems, so that IoT devices following different standards (or no standards...) can
+interoperate.  Therefore, it specifically targets the case where multiple IoT devices
+from different manufacturers and possibly different standards ecosystems need to be used 
+together.
+
+The WoT standards can be considered "essential" when this metadata is not available
+by other means or would be expensive or inconvenient to obtain, or where machine-readable
+metadata is needed to automate integration.
+
+When combined with a browser, the consistent metadata provided by WoT standards
+can be used to automatically build web user interfaces for IoT devices, making them
+more consistently accessible.  This may or may not be done by directly talking to 
+IoT devices.  Very often an IoT system will digest the metadata about a set of
+IoT devices and use this to generate a consistent interface, such as a dashboard.
+
+Some usages of WoT may be safety-critical, for example, an IoT interface to a factory
+actuator or to a fall-detection system in the home.  A government may also use WoT
+to provide access to public services, including critical mechanisms such as access
+to environmental monitoring systems (which might in turn be used to warn vulnerable
+people about air pollution states they need to be wary of). 
+
+The WoT standards primarily describe metadata and how to access it.  To mitigate
+failures, such as the inability to access metadata if a WoT Discovery service
+is unavailable, such systems should use WoT Discovery and metadata (WoT Thing
+Descriptions) for initial connections and if updated metadata is not available,
+use the last known state of this metadata.
+
+### 2. [What kinds of activities could your specification become a part of that you are not designing for?](https://w3ctag.github.io/societal-impact-questionnaire/#unexpected-use)
+WoT does not directly address accessibility but IoT systems generally can be
+used to build custom user interfaces, including those that make systems easier
+to use (e.g. large buttons to replace difficult to use controls) or map sensory
+modalities (e.g. mapping visual indicators to audible ones or vice-versa).
+Unfortunately, the current WoT specification describes only network communication and interactions,
+not physical behavior, making automated construction of such interfaces difficult.
+However, since WoT uses JSON-LD, it would be possible in theory to annotate WoT
+Thing Descriptions with information such as this.
+
+In general, additional metadata can be attached to WoT Thing Descriptions and
+this can be used for arbitrary purposes that can go beyond what is currently
+in the specification.  For example, a given device manufacturer could attach
```

Comment:
I think geolocation should not be called out as a use case the group did not account for.

I think one example would be digital twins - there are other use cases where we identified specification gaps, so these are also candidates for usages where the system is currently not designed for.

Reply:
My use-case about hazards might be mentioned as another.

Reply:
The text point out the option about additional metadata in TDs. The spec gives examples how this can work which also includes geolocation-based metadata examples (also see https://w3c.github.io/wot-thing-description/#semantic-annotations-example-geoloc).